rsts/deployment/deployment/index.rst

```diff
@@ -156,10 +133,13 @@ Deployment Tips and Tricks
 
 Due to the many choices and constraints that you may face in your organization, the specific steps for deploying Flyte
 can vary significantly. For example, which cloud platform to use is typically a big fork in the road for many, and there
-are many choices to make in terms of ingresses, auth providers, and versions of different dependent libraries that
+are many choices to make in terms of Ingress controllers, auth providers, and versions of different dependent libraries that
 may interact with other parts of your stack.
 
-In addition to searching and posting on the `Flyte Slack community <https://flyte-org.slack.com/archives/C01P3B761A6>`__,
+Considering the above, we recommend checking out the `"Flyte The Hard Way" <https://github.com/davidmirror-ops/flyte-the-hard-way/tree/main#flyte-the-hard-way>`__ set of community-maintained tutorials that can guide you through the process of preparing the infrastructure and
```

Comment:
In general I would avoid pointing to an external guide, give also that the guide you are referring to is mostly referring to the single binary deployment, which might be confusing. I think all documentation should be in one place

Reply:
Agree. A second iteration of updates to these guides should be pointed to extend the reach and make it more actionable. Also expanding the tutorial in the https://github.com/unionai-oss/deploy-flyte repo to cover `flyte-core`

Reply:
what if we forked your guide, @davidmirror-ops , and moved it to flyteorg?

Reply:
@eapolinario I was thinking of something similar. Moving forward, the idea is that TF is the preferred approach but a manual guide is always a good resource